### PR TITLE
Implement rich comparison for !=, >=, <=

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

File: f8a_version_comparator/comparable_version.py [hunk 1]
```diff
@@ -75,6 +85,16 @@ def __lt__(self, other):
...
 
+    def __le__(self, other):
+        """Compare ComparableVersion objects.
+
+        This rich comparison implies whether self <= other
+        """
+        if other is None:
+            return False
+
+        return self.compare_to(other) == -1 or self.compare_to(other) == 0
```
Comment (tisnik) [inline f8a_version_comparator/comparable_version.py]: return self.compare_to(other) <= 0

**Review by tisnik — APPROVED** [review]
LGTM, just two nitpicks - how to make the code shorter & simpler by comparing `<= 0` and `>= 0`

File: f8a_version_comparator/comparable_version.py (hunk 1 above)
Comment (msrb) [reply f8a_version_comparator/comparable_version.py]: Good catch, PR updated :wink:

**Review by CermakM — APPROVED** [review]
LGTM :+1:

**Pull request merged**